CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+Your interest in contributing to this project is greatly appreciated. Please know that coding is not the only way to contribute. 
+
+**Here are some ways you can help that doesn't involve coding:**
+
+- Review any active PRs. If the PR needs any changes, please exlpain them in a simple way.
+- You can test the project to see if it has any bugs. If you come across a bug that has not been addressed before, please explain the scenario in a clear and consice manner.
+
+**If coding is your choice of contribution:**
+
+- If you are not sure how to contribute, please contact the Mote modernizing team. 
+- If you find an [issue](https://github.com/fedora-infra/mote/issues) that interests you, comment under it so it can be assigned to you. You can only create a PR for an issue you are assigned to.
+- Please keep the PRs atomic (only one PR per issue)
+
+If this is your first time working with git, below are clear instructions on how to proceed.
+
+a. Create your upstream repo: 
```

Comment:
"Clone the upstream repo" instead of "Create your upstream repo". I don't know, what do you folks think? :thinking: